policy/release/sbom_cyclonedx.rego

```diff
@@ -116,8 +116,9 @@ deny contains result if {
 
 	msg := regex.replace(object.get(property, "value", ""), `(.+)`, ` to "$1"`)
 
+	id := object.get(component, "purl", component.name)
```

Comment:
I see now that `name` and `type` are the only [required properties](https://github.com/enterprise-contract/ec-policies/blob/a9c2e4c0a708e16b00a2e71d4cc3f1d182ede5fb/policy/release/sbom_cyclonedx_schema.rego#L384-L386).